src/blockchain_txn_add_subnetwork_v1.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+package helium;
+
+import "blockchain_token_type_v1.proto";
+
+message blockchain_txn_add_subnetwork_v1 {
+    blockchain_token_type_v1 token_type = 1;
```

Comment:
should the token types supported by the L1 helium network, including HNT and HST be separately tracked from subnetwork token types? This implies that HNT and HST could be valid token types for a subnetwork but that doesn't seem correct.

what i'm suggesting is perhaps two different enumerables, one that includes _all_ token types for purposes like payments and conversions and another for things like subnets where it seems like the top-level L1 tokens aren't in play?

Reply:
Yeah, that's a valid point. However, we can invalidate at txn validity level as well (which is what we're doing right now in the core branch) so not _really_ a huge deal. Also, given the time constraints I don't feel the need to change right away.